code/game/objects/structures/tables_racks.dm

```diff
@@ -294,6 +299,59 @@
 	..()
 	return SECONDARY_ATTACK_CONTINUE_CHAIN
 
+/obj/structure/table/attack_hand_secondary(mob/user, list/modifiers)
+	. = ..()
+	if(!istype(user) || !user.can_interact_with() || isobserver(user))
```

Comment:
Still need the src here. we remove it when pulling vars from ourselves because it's redundant, but if you don't pass anything into this proc then the proc won't have anything to check against
```suggestion
	if(!istype(user) || !user.can_interact_with(src) || isobserver(user))
```